### PR TITLE
Add the HPC High IO with Slurm on GCP v5 example to the integration tests

### PR DESCRIPTION
**Pull request by heyealex**
Additionally redistribute some tests to average out the length

Dependent on #730 

### Submission Checklist

* [x] Have you installed and run this change against pre-commit? (`pre-commit
  install`)
* [x] Are all tests passing? (`make tests`)
* [x] Have you written unit tests to cover this change?
* [x] Is unit test coverage still above 80%?
* [x] Have you updated all applicable documentation?
* [x] Have you followed the guidelines in our Contributing document?

### CODE REVIEW

**Comment by heyealex** [comment]
Manual build of group 5: [build](https://console.cloud.google.com/cloud-build/builds;region=global/25911c37-2efe-44d0-b7cc-bfda4ef2f734)

File: tools/cloud-build/daily-tests/integration-group-1.yaml [hunk 1]
```diff
@@ -16,13 +16,11 @@
 # Current parallel execution tree, built to minimize total execution time and faster tests first.
 # ├── build_ghpc
 # └── fetch_builder
-#    ├── hpc-high-io (group 1)
+#    └── hpc-high-io-v4
+#        └── hpc-high-io-v5
+#           └── batch-mpi
```
Comment (cboneti) [inline tools/cloud-build/daily-tests/integration-group-1.yaml:21]: nit: batch-mpi is no longer here, remove line 21?

Comment (heyealex) [reply tools/cloud-build/daily-tests/integration-group-1.yaml]: Done

File: community/examples/slurm-gcp-v5-high-io.yaml [hunk 2]
```diff
@@ -21,7 +21,10 @@ vars:
   deployment_name: high-io-slurm-gcp-v5
   region: us-central1
   zone: us-central1-c
-  enable_reconfigure: true
+  disable_public_ips: false
```
[the remark this answers is not in the document]
Comment (heyealex) [reply community/examples/slurm-gcp-v5-high-io.yaml:26]: It's being used by `disable_controller_public_ips` and `disable_login_public_ips`. It'd be better to unify these variable names, but I didn't want to pair that with this already fairly big PR.

I have them set to enable (or rather !disabled) to allow access during testing, but also providing an easy way for users to turn off that functionality if needed.

I'll add an extra comment to clarify that.

[the remark this answers is not in the document]
Comment (cboneti) [reply community/examples/slurm-gcp-v5-high-io.yaml:26]: Oh, I get it, my bad, sorry.

**Review by cboneti — APPROVED** [review]
This is great! Thanks!!

**Pull request merged**